site/content/docs/5.3/examples/cheatsheet/index.html

```diff
@@ -202,7 +202,7 @@ <h3>Tables</h3>
           <tr>
             <th scope="row">3</th>
             <td colspan="2">Larry the Bird</td>
```

Comment:
I'd nix Larry the Bird references too since that was the name of Twitter's bird mascot back in the day.